guides/release/components/conditional-content.md

```diff
@@ -148,3 +164,5 @@ avatar, and omit it from the sent message avatar.
     <img src="/images/mascots/zoey.png" role="presentation" alt="">
   </div>
 </div>
+
+Refer the [API doc of the `if` helper](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/if?anchor=if) for more patterns
```

Comment:
do we need to add the nested helper example we've discussed in API doc? @jenweber